src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = getReadFromFile(fromFileName);
+        String report = getReportToFile(dataFromFile);
+        getWriteToFile(report,toFileName);
+    }
+
+    private String getReadFromFile(String fromFileName) {
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder builder = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                builder.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+            return builder.toString();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+    }
+
+    private String getReportToFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        String[] dataSplit = fromFileName.split(" ");
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+
+        for (String word : dataSplit) {
+            String[] arrayDataSplit = word.split(",");
+            if (arrayDataSplit[0].equals("supply")) {
```

Comment:
0 and 1 (array indexes) should be class constants - like ACTION_INDEX, AMOUNT_INDEX